File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/ClaimAdditionalRequiredDocument.cs
namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class ClaimAdditionalRequiredDocument
    {
        public string DocumentName { get; set; }
        public int ClaimAdditionalRequiredDocumentId { get; set; }
        public int PersoneventId { get; set; }
        public int DocumentId { get; set; }
        public bool IsUploaded { get; set; }
        public int DocumentGroupId { get; set; }
        public bool IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }
        public bool VisibletoMember { get; set; }
        public string RequestNote { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/claim-additional-required-document.ts
import { DocumentTypeEnum } from "projects/shared-models-lib/src/lib/enums/document-type.enum";

export class ClaimAdditionalRequiredDocument {
  id: number;
  personeventId: number;
  documentId: number;
  isUploaded: boolean;
  isDeleted: boolean;
  createdBy: string;
  createdDate: Date;
  modifiedBy: string;
  modifiedDate: Date;
  documentGroupId: number;
  documentName: string;
  visibleToMember: boolean;
  requestNote: string;
}
